python/amici/petab_import.py

```diff
@@ -45,43 +47,42 @@ def get_fixed_parameters(
         List of IDs of parameters which are to be considered constant.
     """
 
-    # column names are model parameter names that should be made constant
-    # except for any overridden parameters
+    # Column names are model parameter IDs, compartment IDs or speciesIDs.
+    # Thereof, all parameters except for any overridden ones should be made
+    # constant.
     # (Could potentially still be made constant, but leaving them might
     # increase model reusability)
+
+    # handle parameters in condition table
     if condition_df is not None:
         fixed_parameters = list(condition_df.columns)
+        # get rid of conditionName column
         try:
-            fixed_parameters.remove('conditionName')
+            fixed_parameters.remove(CONDITION_NAME)
         except ValueError:
             pass
-        # remove overridden parameters
+
+        # remove overridden parameters (`object`-type columns)
         fixed_parameters = [p for p in fixed_parameters
-                            if condition_df[p].dtype != 'O']
+                            if condition_df[p].dtype != 'O'
+                            and sbml_model.getParameter(p) is not None]
```

Comment:
this is bc overridees need not be in the sbml, but can also be in the parameter file?